packages/editor-ui/public/n8n-beta-logo-dark-mode.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="201" height="40" viewBox="0 0 201 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
instead of creating new logo files, perhaps we should use `currentColor` to customize the text color.

Reply:
does not seem to work with the img tag.. will require some refactoring to fix